proto/backup.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+package backup;
+
+import "kvrpcpb.proto";
+import "errorpb.proto";
+
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (rustproto.lite_runtime_all) = true;
+
+option java_package = "org.tikv.kvproto";
+
+// The message save the metadata of a backup.
+message BackupMeta {
+    // ID and version of backuped cluster.
+    uint64 cluster_id = 1;
+    string cluster_version = 2;
+
+    // The path where backup data are saved.
+    string path = 3;
+    // A set of files that compose a backup.
+    repeated File files = 4;
+
+    // A pair of timestamp specifies a time range of a backup.
+    // For full backup, the start_version equals to the end_version,
+    // it means point in time.
+    // For incremental backup, the time range is specified as
+    // (start_version, end_version].
+    uint64 start_version = 5;
+    uint64 end_version = 6;
+
+    // Additional metadata describes database and table info.
+    repeated Schema schemas = 7;
+}
+
+message File {
+    string name = 1;
+    uint32 crc32 = 2;
+
+    bytes start_key = 3;
+    bytes end_key = 4;
+    uint64 start_version = 5;
+    uint64 end_version = 6;
+}
+
+message Schema {
+    bytes db = 1;
+    bytes table = 2;
```

Comment:
Is table contain how to create the table?

Reply:
Yes, it contains model.tableinfo. https://github.com/pingcap/parser/blob/8d44c03502f6b50e6111101c5504488d17f411b3/model/model.go#L194